### ISSUE TITLE
0012-Philosophy Reference sequence didn't support in output

### ISSUE THREAD

**Issue by srm-openstax** [openstax/cnx-recipes#4239]
We have moved the references to the respective modules as per the customer instruction. But the alphabetical order sequence issue appeared in the output. Kindly check and resolve this issue.

![Philosophy_Reference](https://user-images.githubusercontent.com/86622862/142873803-37136f27-94a4-4bf0-866d-0aa28edfa1c3.png)

# Bug Location(s)
collection ID: col00005
server: https://coffee-grouse-ulafn81g.ws-us18.gitpod.io/
chapter/section: Chapter 1/Section 1-4
surrounding text: Adler, Joseph A. 2006. “Daughter/Wife/Mother or

**Comment by lwtchu** [comment]
# Acceptance Criteria

Is it possible to alphabetize these references? I think they've been done in the past.

**Comment by ajluk** [comment]
@lwtchu As I checked references hasn't been alphabetized before in kitchen, they were just sorted alphabetically in sections in the content and divided through baking by sections titles while moved into eoc, so it looked as they were sorted in kitchen. In Philosophy the references sections were just moved to eoc as a whole without dividing them into sections by titles.  When I added section titles and divide them into sections they look like this: 
![image](https://user-images.githubusercontent.com/55203598/144591596-9bf2a9dd-ad00-42db-8e40-ddec9424c608.png)
I started working on sorting references alphabetically in kitchen, but let me know if this something that should be implemented (move all references sections to eco and sort them alphabetically without dividing them by section title)? :slightly_smiling_face:
